quarkchain/cluster/cluster_config.py

```diff
@@ -167,6 +167,15 @@ class MonitoringConfig(BaseConfig):
     ERRORS = "error"
 
 
+class PrometheusConfig(BaseConfig):
+    TIMEOUT = 10
+    HOST = "http://localhost:38391"
+    GAP = 30  # Interval between two total_balance queries
+    TOKEN = 0x8BB0
```

Comment:
we may want to support a list of tokens to watch. also, using token string seems easier to read than the number